egs/bentham/v1/local/create_splits.sh

```diff
@@ -28,9 +28,9 @@ function split {
         echo $name $spkid >> $split_dir/utt2spk 
 	done < "$line_file"
     
-    sed -i '/^\s*$/d' $split_dir/images.scp
-    sed -i '/^\s*$/d' $split_dir/text
-    sed -i '/^\s*$/d' $split_dir/utt2spk
+    grep -v -e '^\s*$' $split_dir/images.scp
+    grep -v -e '^\s*$' $split_dir/text
+    grep -v -e '^\s*$' $split_dir/utt2spk
```

Comment:
The intent here is to delete whitespace-only lines, again, in a file in-place. The typical rerl trick for that is to output non-whitespace instead, like this:
```
perl -i -ne 'print if /\S/' $split_dir/images.scp $split_dir/text $split_dir/utt2spk
```
The `\S` stands for "any non-space character" (like any not matched by the lowercase `\s`). Thus, any line with a non-space is preserved.